wgpu/Cargo.toml

```diff
@@ -76,9 +76,10 @@ name = "water"
 test = true
 
 [features]
-default = []
+default = ["wgsl"]
 spirv = ["naga/spv-in"]
```

Comment:
I'm skeptical of default features here - imo default features are impossible to turn off once you have multiple crates in your tree that depend on the crate - as someone at some point has forgotten to turn them off. This is something we should chat about though, as I'm sure some people have opinions on it.

Reply:
I agree with you, I only added it to keep the behavior the same and any breaking changes to a minimum, will remove it.

Reply:
I think it would be confusing to require a feature to use WGSL because it's the shading language for WebGPU, and almost all crates will enable this feature anyway.

Reply:
I removed it for now, happy to change it to whatever you guys end up deciding on. Also want to note that if we decide removing this as a default crate feature, we might want to add more tests with different combinations in CI?

Reply:
I think I agree with groves here on second thought. Let's keep wgsl as a default feature - it's going to be a footgun if you can't use wgpu without any features.

Reply:
Done.

Reply:
I just noticed, should I add the `wgsl` feature as a default to `wgpu-core` too?

Reply:
No, that's fine - wgpu-core is essentially an implementation detail.